app/controllers/cities_controller.rb

```diff
@@ -22,7 +22,12 @@ def forbes_index
   end
 
   def forbes_show
-    use_new_styles
+    if current_user
+      use_new_styles
+    else
+      flash[:notice] = 'You should sign up first!'
```

Comment:
@ankitshah811 to provide copy for this. just comment with it here and i'll change this.